frontend/public/modules/photos/views/view-photo.client.view.html

```diff
@@ -1,3 +1,11 @@
 <section data-ng-controller="PhotosController" data-ng-init="findOne()">
+<<<<<<< HEAD
```

Comment:
Got it. Will clean this up at lunch. 

Jen

jtrudell@gmail.com 
312-218-6381 (cell)
jtrudell78 (Skype)

> On Oct 12, 2015, at 6:50 AM, Brian Turnbull <notifications@github.com> wrote:
> 
> In frontend/public/modules/photos/views/view-photo.client.view.html:
> 
> > @@ -1,3 +1,11 @@
> >  <section data-ng-controller="PhotosController" data-ng-init="findOne()">
> > +<<<<<<< HEAD
> You have a merge conflict that needs to be fixed here.
> 
> —
> Reply to this email directly or view it on GitHub.
>